### PR TITLE
Fix validation message on triggers wizard

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Correct validation message in New Compliance Trigger](https://trello.com/c/9Ogvnmx6/664-correct-validation-message-in-new-compliance-trigger)

File: new-lamassu-admin/src/pages/Triggers/Wizard.js [hunk 1]
```diff
@@ -236,21 +272,28 @@ const Wizard = ({ onClose, save, error, currency }) => {
         />
         <Formik
           validateOnBlur={false}
-          validateOnChange={false}
+          validateOnChange={true}
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Triggers/Wizard.js:275]: Maybe this should remain false?

File: new-lamassu-admin/src/pages/Triggers/helper.js [hunk 2]
```diff
@@ -223,18 +223,48 @@ const typeSchema = Yup.object()
...
+        txAmount: threshold => threshold.threshold > 0,
         txVolume: threshold =>
-          threshold.threshold >= 0 && threshold.thresholdDays >= 0,
+          threshold.threshold >= 0 && threshold.thresholdDays > 0,
         txVelocity: threshold =>
-          threshold.threshold >= 0 && threshold.thresholdDays >= 0,
-        consecutiveDays: threshold => threshold.thresholdDays >= 0
+          threshold.threshold > 0 && threshold.thresholdDays > 0,
+        consecutiveDays: threshold => threshold.thresholdDays > 0
       }
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Triggers/helper.js:260]: I was thinking about doing similar custom messages for the validation when editing triggers, but to be consistent with the rest of the code I validated each field individually and used the default yup messages. You can see that here https://github.com/lamassu/lamassu-server/pull/657 and use this for reference https://github.com/jquense/yup/blob/91ace1e8be3fc23c775ec8117c47b406bf29da4a/src/locale.ts. So maybe we should go towards the same approach. Can you ask Rafael about this? he will know better.

File: new-lamassu-admin/src/pages/Triggers/Wizard.js (hunk 1 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/pages/Triggers/Wizard.js:275]: Having this on false, would make the form much less responsive to user interaction

Comment (josepfo) [reply new-lamassu-admin/src/pages/Triggers/Wizard.js:275]: yeah but I once worked on a card where I was supposed to force the forms validation only after submit. Check this with Rafael if you can.

**Pull request closed without merging**